omod/src/main/java/org/openmrs/module/appointments/web/contract/AppointmentRequest.java

```diff
@@ -126,4 +127,12 @@ public List<AppointmentProviderDetail> getProviders() {
     public void setProviders(List<AppointmentProviderDetail> providers) {
         this.providers = providers;
     }
+
+    public String getPriority() {
+        return priority;
+    }
+
+    public void setPriority(String priority) {
```

Comment:
Why are getters/setters for Appointment Priority using String, instead of Enum?

Reply:
This is request contract class and we followed the same as that of appointment status

Reply:
If we are to change it from String to Enum now, it would require some more time and we need to change it accordingly to the FHIR codes of priority. So we created a card in the backlog to track it [BAH-3637](https://bahmni.atlassian.net/browse/BAH-3637) and will be taken up later.